pkg/amqp-tools/ConnectionConfig.php

```diff
@@ -226,29 +229,37 @@ public function getVHost()
     }
 
     /**
-     * @return int
+     * @return float
      */
     public function getReadTimeout()
     {
         return $this->getOption('read_timeout');
     }
 
     /**
-     * @return int
+     * @return float
      */
     public function getWriteTimeout()
     {
         return $this->getOption('write_timeout');
     }
 
     /**
-     * @return int
+     * @return float
      */
     public function getConnectionTimeout()
     {
         return $this->getOption('connection_timeout');
     }
 
+    /**
+     * @return float
+     */
+    public function getChannelRpcTimeout()
```

Comment:
This is a php-amqplib specific option, I don't think we should introduce a dedicated getter for it in amqp tools. 

Could you use getOption method to get its value ?

Reply:
Updated to limit code changes to `pkg/amqp-lib`.